section4/exercises/burrito.rb

```diff
@@ -1,19 +1,41 @@
-# Add the following methods to this burrito class and 
+# Add the following methods to this burrito class and
 # call the methods below the class:
 # 1. add_topping
 # 2. remove_topping
 # 3. change_protein
 
 class Burrito
   attr_reader :protein, :base, :toppings
-  def initialize(protein, base, toppings)
-    @protein  = protein
-    @base     = base
-    @toppings = toppings
+  def initialize(p, b, t)
```

Comment:
It is better Ruby convention to use full words for variables names. This makes it easier for the reader (and you) to discern what they are.